docs/advanced_installation.rst

```diff
@@ -14,7 +14,7 @@ pip
 ---
 
 We always recommend installing in a virtual environment such as Conda.
-If not, then use ``pip3``, ``python3`` below.
```

Comment:
I'm okay with python/pip without the 3. But we should be consistent. Later python3 is used in a couple of places (when running the calling script). We should decide generally for the docs whether we want to drop the 3.